bokeh/models/renderers.py

```diff
@@ -242,6 +242,18 @@ def _check_malformed_graph_source(self):
     rendered as the graph edges.
     """)
 
+    selection_mode = Enum("default", "linked", help="""
+    With the "default" selection mode, node glyphs will be selected in the same
+    way as normal glyphs. With "linked" mode, selecting a node will also select
+    the attached edges.
+    """)
+
+    inspection_mode = Enum("default", "linked", help="""
+    With the "default" inspection mode, node glyphs will be inspected in the same
+    way as normal glyphs. With "linked" mode, inspecting a node will also inspect
+    the attached edges.
+    """)
```

Comment:
Also we should check with @hsparra but I think in a default "linked" mode, the desire might also be for adjacent nodes to get selected, not just the edges leading to them.